libfreerdp/CMakeLists.txt

```diff
@@ -89,20 +89,18 @@ if (WITH_CAIRO)
     find_package(Cairo REQUIRED)
 endif(WITH_CAIRO)
 
-if (SWScale_FOUND)
+# Prefer SWScale over Cairo, both at the same time are not possible.
+if (WITH_SWSCALE)
```

Comment:
By far not a cmake expert but shouldn't we still check for `SWScale_FOUND` and only set `SWSCALE_FOUND=1` if both swscale was found and `WITH_SWSCALE` is enabled (same for cairo)?

Reply:
@mfleisz yes, look 4 lines above ;)

Reply:
@akallabeth Yes that's where we search for swscale ... but if I define `WITH_SWSCALE` then `-DSWSCALE_FOUND=1` below is defined ... no matter if the library was found or not ... or am I missing something 😅 ?

Reply:
@mfleisz it is defined as `CMake` variable, not for `C` preprocessor symbol